external-crates/move/crates/move-analyzer/trace-adapter/src/trace_utils.ts

```diff
@@ -186,36 +202,59 @@ interface JSONTraceCloseFrame {
     return_: JSONTraceRuntimeValueContent[];
 }
 
-interface JSONExternalMoveCallInfo {
+interface JSONExtMoveCallSummary {
```

Comment:
Unrelated renamings to shorten up the full names a bit (much like in the runtime)